algorithm_catalog/bap_composite_files/bap_composite_description.md

```diff
@@ -88,4 +88,4 @@ White, J.C.; Wulder, M.A.; Hobart, G.W.; Luther, J.E.; Hermosilla, T.; Griffiths
 Residual cloud artifacts may be present in the composite, especially for smaller time windows or during cloudy seasons. The cloud artifacts are caused by the limited capabilities of the default Sentinel-2 cloud detection mechanism to correctly identify all clouds. 
 The picture below shows a composite image with some cloud shadow pixels remaining. Because these pixels were not classified as cloud shadow in the SCL band, they are not masked out in the final composite.  
 
-![cloud-shadow-artifacts-bap.png](./cloud-shadow-artifacts-bap.png)
\ No newline at end of file
+![cloud-shadow-artifacts-bap.png](https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/main/algorithm_catalog/bap_composite_files/cloud-shadow-artifacts-bap.png)
```

Comment:
Actually at the moment we have added a logic to build this image url automatically, but the output is not always working. This is because there is a case where the image directory name doesn't have the same structure with the rest of the UDP here https://github.com/ESA-APEx/apex_algorithms/tree/main/algorithm_catalog/sentinel1_stats (in this PR, I have renamed the folder). If we can make this consistent, surely we don't need to rewrite the image url directly on the description.